karma.conf.js

```diff
@@ -103,39 +54,30 @@ module.exports = function(config) {
 
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ['mocha', 'sinon', 'chai', 'sinon-chai'],
-
+    frameworks: ['requirejs', 'mocha', 'chai'],
```

Comment:
yeah, a few commits fail the tests, but I wanted to make it easier to review each piece